keyboards/ramonimbao/herringbone/pro/keymaps/via/keymap.c

```diff
@@ -76,6 +76,27 @@ void matrix_scan_user(void) {
     }
 }
 
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
```

Comment:
As this issue can extend to other extra keys, would it not be cleaner to just add a `wait_ms(20)` to encoder_update_user?

Reply:
From testing, it only really happens with the volume keys. But you're right that is a cleaner way to do it.